celest/angle_strings.py

```diff
@@ -0,0 +1,62 @@
+
+
+from celest.units.quantity import Quantity
+from celest import units as u
+
+
+DEGREE_SYMBOL = "\u00B0"
+MINUTE_SYMBOL = "\u2032"
+SECOND_SYMBOL = "\u2033"
+
+
+def _ISO6709_representation(latitude: Quantity, longitude: Quantity, height:
+                            Quantity) -> str:
+    """Format geographical data in the ISO6709 standard.
+
+    Parameters
+    ----------
+    latitude : Quantity
+        Latitude angle in decimal degrees.
+    longitude : Quantity
+        Longitude angle in decimal degrees.
+    height : Quantity
+        Altitude in km.
+
+    Returns
+    -------
+    str
+        ISO6709 standardized position string.
+    """
+
+    latitude_string = _get_latitude_string(latitude)
+    longitude_string = _get_longitude_string(longitude)
+    altitude_string = _get_altitude_string(height)
+
+    return latitude_string + " " + longitude_string + " " + altitude_string
```

Comment:
Would something like `" ".join([_get_latitude_string(latitude), _get_longitude_string(longitude), _get_altitude_string(height)])` be cleaner?